### ISSUE TITLE
[Visual Requirement -Embedded-Create Profile]:  The Luminosity ratio of the "Create Profile", "Account Information" and "Done" button is less than 4.5:1.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#22]
Priority: Medium

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Verify that the luminosity ratio of the ""Create Profile"" page   is less than 4.5:1.

Note:
1. Luminosity ratio is less than 4.5:1 for many other different other elements texts and icons. (see attachment)

Expected
The luminosity ratio of the  ""Create Profile"", ""Account Information"" and ""Done"" button should be greater than or equal to 4.5:1.

Actual
The luminosity ratio of the ""Create Profile"", ""Account Information"" and ""Done""  button is less than 4.5:1.

Note : 
1> Same issue is repro throughout application for this color.

MAS Rule: MAS1.4.3

Impact
User who have vision Issues will find difficulty as Luminosity ratio is less than 4.5:1

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : CCA
